test/run_nighthawk_bazel_coverage.sh

```diff
@@ -41,7 +41,7 @@ COVERAGE_VALUE=${COVERAGE_VALUE%?}
 
```

Comment:
Can we (for now) add a TODO pointing to an issue about the missing test coverage? Not sure if this is the right file, so feel free to add it in the appropriate location.